src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write code here
+const liList = document.querySelectorAll('li');
+
+for (const item of liList) {
+  const newSpan = document.createElement('span');
+
+  newSpan.append(item.firstChild);
+  item.prepend(newSpan);
+
+  newSpan.addEventListener('click', (e) => {
+    if (e.target.nextSibling) {
+      e.target.nextSibling.hidden = !e.target.nextSibling.hidden;
```

Comment:
Remember that by setting hidden on the elements they are still in the dom, if you would like to remove them completely you would have to use display: 'none'